internal-include/darlingserver/process.hpp

```diff
@@ -170,6 +171,9 @@ namespace DarlingServer {
 		std::shared_ptr<Process> tracerProcess() const;
 		bool setTracerProcess(std::shared_ptr<Process> tracerProcess);
 
+		const std::string& executablePath() const;
```

Comment:
We don't want to return a reference to data that is protected by a lock.
```suggestion
		std::string executablePath() const;
```